plugins/ccgx/fu-ccgx-hpi-device.c

```diff
@@ -348,11 +309,29 @@ fu_ccgx_hpi_device_i2c_write (FuCcgxHpiDevice *self,
 		g_prefix_error (error, "i2c write error: bulk xfer: ");
 		return FALSE;
 	}
-
 	/* 10 msec delay */
 	g_usleep (I2C_READ_WRITE_DELAY_US);
-	if (!fu_ccgx_hpi_device_wait_for_notify (self, NULL, error)) {
-		g_prefix_error (error, "i2c wait for notification error: ");
+	
+	i2c_status[0] = 0;
```

Comment:
Is this actually necessary, shouldn't it have always been updated by the below interrupt transfer?

Reply:
I will remove redundent "i2c_status[0] = 0;".

Reply:
I fixed it.